loopy/target/cuda.py

```diff
@@ -312,6 +313,12 @@ def known_callables(self):
         callables.update(get_cuda_callables())
         return callables
 
+    def symbol_manglers(self):
+        from loopy.target.opencl import opencl_symbol_mangler
+        return (
+                super().symbol_manglers() + [
+                    opencl_symbol_mangler
```

Comment:
Please add a comment in `opencl_symbol_mangler` that it is also being used in CUDA, otherwise someone might inadvertently break compatibility.

Reply:
Done !